src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/commands.py

```diff
@@ -153,17 +153,17 @@ def load_command_table(self, _):
                               'restore_storage_account',
                               doc_string_source=data_doc_string.format('restore_storage_account'))
 
-    with self.command_group('keyvault storage sas-definition', kv_data_sdk) as g:
-        g.keyvault_command('create',
-                           'set_sas_definition',
-                           doc_string_source=data_doc_string.format('set_sas_definition'))
-        g.keyvault_command('list', 'get_sas_definitions')
-        g.keyvault_command('show', 'get_sas_definition')
-        g.keyvault_command('update',
-                           'update_sas_definition',
-                           doc_string_source=data_doc_string.format('update_sas_definition'))
-        g.keyvault_command('delete', 'delete_sas_definition')
-        if data_api_version != '2016_10_01':
+    if data_api_version != '2016_10_01':
+        with self.command_group('keyvault storage sas-definition', kv_data_sdk) as g:
```

Comment:
Same here. The command_group call should include the min_api kwarg to effect this change.